[2024-09-09T21:11:33.638+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_daily_order_update_dag.dbt_run manual__2024-09-09T21:11:10.637208+00:00 [queued]>
[2024-09-09T21:11:33.642+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_daily_order_update_dag.dbt_run manual__2024-09-09T21:11:10.637208+00:00 [queued]>
[2024-09-09T21:11:33.642+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-09T21:11:33.648+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): dbt_run> on 2024-09-09 21:11:10.637208+00:00
[2024-09-09T21:11:33.651+0000] {standard_task_runner.py:57} INFO - Started process 359 to run task
[2024-09-09T21:11:33.654+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'run_daily_order_update_dag', 'dbt_run', 'manual__2024-09-09T21:11:10.637208+00:00', '--job-id', '274', '--raw', '--subdir', 'DAGS_FOLDER/run_daily_order_update.py', '--cfg-path', '/tmp/tmpm4otzkk1']
[2024-09-09T21:11:33.656+0000] {standard_task_runner.py:85} INFO - Job 274: Subtask dbt_run
[2024-09-09T21:11:33.685+0000] {task_command.py:415} INFO - Running <TaskInstance: run_daily_order_update_dag.dbt_run manual__2024-09-09T21:11:10.637208+00:00 [running]> on host 2c451b520589
[2024-09-09T21:11:33.732+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_daily_order_update_dag' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T21:11:10.637208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-09T21:11:10.637208+00:00'
[2024-09-09T21:11:33.732+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T21:11:33.733+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec dbt_container dbt run --project-dir /opt/dbt_vne/dbt_vne/']
[2024-09-09T21:11:33.737+0000] {subprocess.py:86} INFO - Output:
[2024-09-09T21:11:34.540+0000] {subprocess.py:93} INFO - [0m21:11:34  Running with dbt=1.8.6
[2024-09-09T21:11:34.715+0000] {subprocess.py:93} INFO - [0m21:11:34  Registered adapter: postgres=1.8.2
[2024-09-09T21:11:34.778+0000] {subprocess.py:93} INFO - [0m21:11:34  Unable to do partial parsing because of a version mismatch
[2024-09-09T21:11:35.365+0000] {subprocess.py:93} INFO - [0m21:11:35  [[33mWARNING[0m]: Did not find matching node for patch with name 'src_Amz_Orders' in the 'models' section of file 'models/01_src/schema.yml'
[2024-09-09T21:11:35.457+0000] {subprocess.py:93} INFO - [0m21:11:35  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-09-09T21:11:35.457+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2024-09-09T21:11:35.457+0000] {subprocess.py:93} INFO - - models.dbt_vne.04_dims
[2024-09-09T21:11:35.552+0000] {subprocess.py:93} INFO - [0m21:11:35  Found 22 models, 6 sources, 417 macros
[2024-09-09T21:11:35.553+0000] {subprocess.py:93} INFO - [0m21:11:35
[2024-09-09T21:11:36.450+0000] {subprocess.py:93} INFO - [0m21:11:36  Concurrency: 4 threads (target='dev')
[2024-09-09T21:11:36.459+0000] {subprocess.py:93} INFO - [0m21:11:36
[2024-09-09T21:11:36.589+0000] {subprocess.py:93} INFO - [0m21:11:36  1 of 6 START sql table model dev_steve.fct_Amz_Removal_Report .................. [RUN]
[2024-09-09T21:11:36.589+0000] {subprocess.py:93} INFO - [0m21:11:36  2 of 6 START sql table model dev_steve.fct_Amz_ReturnsReport ................... [RUN]
[2024-09-09T21:11:36.613+0000] {subprocess.py:93} INFO - [0m21:11:36  3 of 6 START sql incremental model dev_steve.fct_Amz_Order_Items ............... [RUN]
[2024-09-09T21:11:36.685+0000] {subprocess.py:93} INFO - [0m21:11:36  4 of 6 START sql table model dev_steve.fct_VneAmzPurchasingOrdersDetail ........ [RUN]
[2024-09-09T21:11:37.122+0000] {subprocess.py:93} INFO - [0m21:11:37  2 of 6 OK created sql table model dev_steve.fct_Amz_ReturnsReport .............. [[32mSELECT 48[0m in 0.52s]
[2024-09-09T21:11:37.125+0000] {subprocess.py:93} INFO - [0m21:11:37  4 of 6 OK created sql table model dev_steve.fct_VneAmzPurchasingOrdersDetail ... [[32mSELECT 1248[0m in 0.43s]
[2024-09-09T21:11:37.132+0000] {subprocess.py:93} INFO - [0m21:11:37  1 of 6 OK created sql table model dev_steve.fct_Amz_Removal_Report ............. [[32mSELECT 48[0m in 0.52s]
[2024-09-09T21:11:37.185+0000] {subprocess.py:93} INFO - [0m21:11:37  3 of 6 OK created sql incremental model dev_steve.fct_Amz_Order_Items .......... [[32mINSERT 0 35[0m in 0.57s]
[2024-09-09T21:11:37.186+0000] {subprocess.py:93} INFO - [0m21:11:37  5 of 6 START sql table model dev_steve.ana_sales_detail_last_60_days ........... [RUN]
[2024-09-09T21:11:37.195+0000] {subprocess.py:93} INFO - [0m21:11:37  6 of 6 START sql table model dev_steve.ana_stock ............................... [RUN]
[2024-09-09T21:11:37.585+0000] {subprocess.py:93} INFO - [0m21:11:37  5 of 6 OK created sql table model dev_steve.ana_sales_detail_last_60_days ...... [[32mSELECT 1355[0m in 0.40s]
[2024-09-09T21:11:37.593+0000] {subprocess.py:93} INFO - [0m21:11:37  6 of 6 OK created sql table model dev_steve.ana_stock .......................... [[32mSELECT 258[0m in 0.39s]
[2024-09-09T21:11:37.864+0000] {subprocess.py:93} INFO - [0m21:11:37
[2024-09-09T21:11:37.865+0000] {subprocess.py:93} INFO - [0m21:11:37  Finished running 5 table models, 1 incremental model in 0 hours 0 minutes and 2.31 seconds (2.31s).
[2024-09-09T21:11:37.901+0000] {subprocess.py:93} INFO - [0m21:11:37
[2024-09-09T21:11:37.901+0000] {subprocess.py:93} INFO - [0m21:11:37  [32mCompleted successfully[0m
[2024-09-09T21:11:37.901+0000] {subprocess.py:93} INFO - [0m21:11:37
[2024-09-09T21:11:37.902+0000] {subprocess.py:93} INFO - [0m21:11:37  Done. PASS=6 WARN=0 ERROR=0 SKIP=0 TOTAL=6
[2024-09-09T21:11:38.871+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-09T21:11:38.901+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=run_daily_order_update_dag, task_id=dbt_run, execution_date=20240909T211110, start_date=20240909T211133, end_date=20240909T211138
[2024-09-09T21:11:38.960+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-09T21:11:38.989+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
